css-pseudo-4/Overview.bs

```diff
@@ -724,10 +725,24 @@ Default UA Styles</h3>
     /* Represent default spelling/grammar error styling in an adjustable way */
     :root::spelling-error { text-decoration-line: spelling-error; }
     :root::grammar-error  { text-decoration-line: grammar-error; }
-    /* Highlight targeted text */
-    :root::target-text    { color: MarkText; background: Mark; }
   </pre>
 
+  Some highlight pseudo-elements
+  should have <dfn>default highlight colors</dfn>,
+  consisting of a 'color' and a 'background-color'.
```

Comment:
```suggestion
  Some [=highlight pseudo-elements=]
  have <dfn>paired default highlight colors</dfn>,
  consisting of a default 'color' and a 'background-color'
  that are either used or overridden together,
  see [[#paired-cascade]].
```

Reply:
done in 4e092c568e8a4247309ffff98fae7b172f2b4067